l10n_it_corrispettivi/models/account.py

```diff
@@ -43,12 +43,12 @@ def onchange_company_id_corrispettivi(self):
 
     @api.onchange('partner_id')
     def onchange_partner_id_corrispettivi(self):
-        if not self.partner_id or not self.partner_id.use_corrispettivi:
-            # If partner is not set or its use_corrispettivi flag is disabled,
-            # do nothing
-            return
-
-        self.set_corr_journal()
+        # If partner is not set or its use_corrispettivi flag is disabled,
+        # set default journal, else set default
```

Comment:
@GSLabIt puoi solo rendere più chiaro il commento? `else set default` a cosa si riferisce?

Reply:
@eLBati in effetti era scritto con i piedi, corretto. Ho aggiunto anche il controllo del flag `corrispettivi` sulla posizione fiscale